linguatec_lexicon/admin.py

```diff
@@ -2,24 +2,48 @@
 
 from . import models
 
+
 @admin.register(models.Lexicon)
 class LexiconAdmin(admin.ModelAdmin):
     list_display = ('name', 'src_language', 'dst_language',)
+    search_fields = ('name',)
+    list_filter = ('src_language', 'dst_language',)
+
 
 @admin.register(models.GramaticalCategory)
 class GramaticalCategoryAdmin(admin.ModelAdmin):
     list_display = ('abbreviation', 'title',)
 
+
 @admin.register(models.Word)
 class WordAdmin(admin.ModelAdmin):
     list_display = ('term', 'lexicon',)
+    search_fields = ('term',)
+    list_filter = ('lexicon__name', 'lexicon__src_language', 'lexicon__dst_language',)
 
 
 @admin.register(models.Entry)
 class EntryAdmin(admin.ModelAdmin):
-    list_display = ('translation', 'word',)
+    list_display = ('word', 'translation', 'variation')
+    search_fields = ('word__term',)
+    list_filter = ('word__lexicon', 'word__lexicon__src_language',
+                   'word__lexicon__dst_language', 'variation__region__name')
+
```

Comment:
* The filter 'word__lexicon' does not appear in filter list.
* The filter 'variation__region__name' appears as "By name" filter, could this text be more specific?

Reply:
About the filter 'variation__region__name', I have added a new function which allows to change the name in a easy way, so now the title of the list is by Region instead of by name, thank you for the advice.

About 'word__lexicon' not appearing it is strange because it appears in my project and in my admin panel so i dont know what can be the problem.

![image](https://user-images.githubusercontent.com/64277128/100552054-02909100-3285-11eb-9c61-239046cb4f8e.png)

Reply:
I only had one lexicon, adding another one, the filter appears. But it's quite weird because I have no regions and region filter is visible.

Reply:
I think this is happening because of how the method filter_list behaves when is used with a foreign key variable that can be null, and in our case the entries which do not have a specific variation their variation field take a null/none value, so I have been searching for function that could solve this problem removing the region list when there are no values in the list but I could not have found one.

About the fact that the filter lexicon does not appear unless there are more than one lexicon added in the database, I don't know why it behaves like that, maybe because having only one lexicon in the database which can't be a null/none variable so having it makes it redundant. Although, if this was the case neither src_language nor dst_language should appear.

What do you think? Do you know a method that could solve this?

Reply:
I think that this behaviour is the django defaults. So, no need to pay attention on changing that.